crates/cheatnet/tests/starknet/forking.rs

```diff
@@ -0,0 +1,59 @@
+use crate::assert_success;
```

Comment:
We are lacking tests which check the state differences between different block heights